main.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/kklipsch/cli"
+)
+
+func main() {
+	err := NewKay().Run(os.Args)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err.Error())
+		os.Exit(1)
+	}
+
+	os.Exit(0)
+}
```

Comment:
For some reason this seems more "you":
```
func main() {
	err := NewKay().Run(os.Args)
	
	exitCode := 0
	if err != nil {
		fmt.Fprintln(os.Stderr, err.Error())
		exitCode = 1
	}

	os.Exit(exitCode)
}
```

But perhaps instead just:
```
func main() {
	err := NewKay().Run(os.Args)
	
	if err != nil {
		log.Fatal(err) // "log" uses STDERR by default
	}
}
```
http://golang.org/pkg/log/#Fatal